src/jit/codegenarmarch.cpp

```diff
@@ -1207,7 +1207,34 @@ void CodeGen::genMultiRegCallStoreToLocal(GenTreePtr treeNode)
     {
         // Right now the only enregistrable multi-reg return types supported are SIMD types.
         assert(varTypeIsSIMD(treeNode));
-        NYI("GT_STORE_LCL_VAR of a SIMD enregisterable struct");
+
+        regNumber dst = treeNode->gtRegNum;
+
+        for (unsigned i = 0; i < regCount; ++i)
+        {
+            var_types type = pRetTypeDesc->GetReturnRegType(i);
+            regNumber reg  = call->GetRegNumByIdx(i);
+            if (op1->IsCopyOrReload())
```

Comment:
Looking at this, I was wondering where the actual copy or reload was getting generated. I don't see any code to handle that in `genConsumeRegs()`, but perhaps I've missed something.
Looking at the code for amd64/ux raises the same question for me.
I'm going to look into this a bit further, and will open a separate issue if needed.

Reply:
I wasn't sure either.  I was originally going to mark it NYI, but since a direct copy for the non enregistered case was a direct copy, I just copied the code.